onnxruntime/core/optimizer/transpose_optimizer/transpose_optimizer.cc

```diff
@@ -2032,10 +2036,11 @@ OptimizeResult OptimizeImpl(OptimizerCtx& ctx) {
 
 const std::unordered_set<std::string_view>& GetLayoutSensitiveOps() {
   // List of all layout sensitive ops defined in ONNX standard.
-  static std::unordered_set<std::string_view> layout_sensitive_ops = {"Conv", "QLinearConv", "BatchNormalization",
-                                                                      "AveragePool", "GlobalAveragePool", "MaxPool",
-                                                                      "GlobalMaxPool", "LRN", "GridSample",
-                                                                      "DepthToSpace", "SpaceToDepth"};
+  static std::unordered_set<std::string_view> layout_sensitive_ops = {
+      "Conv", "QLinearConv", "BatchNormalization",
+      "AveragePool", "GlobalAveragePool", "MaxPool",
```

Comment:
MaxUnpool is still a layout sensitive operator isn't it, so should it be in this list even if we don't handle it currently?

Reply:
Done.